pkg/storaged/stratis-utils.js

```diff
@@ -60,3 +68,14 @@ export function get_unused_keydesc(client, desc_prefix) {
                 return desc;
             });
 }
+
+export function confirm_tang_trust(url, adv, action) {
+    dialog_open({
+        Title: _("Verify key"),
+        Body: <TangKeyVerification url={url} adv={adv} />,
+        Action: {
+            Title: _("Trust key"),
+            action
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)